gate-web/gate-web.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
   compile spinnaker.dependency("korkWeb")
   compile spinnaker.dependency("frigga")
   compile spinnaker.dependency('cglib')
-  compile('org.springframework.session:spring-session-data-redis:1.0.1.RELEASE')
+  compile('org.springframework.session:spring-session-data-redis:1.1.1.RELEASE')
```

Comment:
this lets us get rid of some dirty hackery we had to make spring-session-data-redis work with elasticache (see below for details)